docs/core-concepts/dotnet-core.md

```diff
@@ -0,0 +1,145 @@
+.NET Core
+=========
+
+.NET Core is a general purpose development platform maintained by Microsoft and the .NET community on [GitHub](https://github.com/dotnet/core). It is cross-platform, supporting Windows, OS X and Linux and can be used in device, cloud and embedded/IoT scenarios. The supported Operating Systems (OS), CPUs and application scenarios will grow over time, provided by Microsoft, other companies and individuals.
+
+The following characteritics best define .NET Core:
+
+- **Flexible deployment:** app-local or side-by-side user- or machine-wide installation.
+- **Cross-platform:** Runs on Windows, OS X and Linux; Can be ported to other OSes.
+- **Compatibility:** .NET Core is compatible with .NET Framework and Xamarin platforms.
+- **Open source:** The .NET Core platform is open source, using MIT and Apache 2 licenses.
+
+Composition
+===========
+
+.NET Core is composed of a few parts:
+
+- A [.NET runtime](https://github.com/dotnet/coreclr), which provides a type system, assembly loading, a garbage collector, native interop and other basic services. 
+- A set of [framework libraries]((https://github.com/dotnet/corefx)), which provide primitive data types, app composition types and fundamental utilities. 
+- The 'dotnet' app host, which is used to launched .NET Core apps. It selects the runtime and hosts the runtime, provides an assembly loading policy and launches the app. The same host is also used to launch SDK tools in much the same way.
+- A [set of SDK tools](https://github.com/dotnet/cli) and language compilers that enable the base developer experience, available in the [.NET Core SDK](http://dotnet.github.io/docs/core-concepts/core-sdk).
+
+Languages
+---------
+
+The C# languages (F# and VB are coming) can be used to write applications and libraries for .NET Core. The compilers run on .NET Core, enabling you to develop for .NET Core anywhere it runs. In general, you will not use the compilers directly, but indirectly by using the SDK tools.
+
+The C# Roslyn compiler and the .NET Core tools have been integrated into several text editors and IDEs, including Visual Studio, Visual Studio Code, Sublime Text and Vim, making .NET Core development an option in your favorite coding environment and OS. This integration is provided, in part, by the good folks of the [OmniSharp project](http://www.omnisharp.net/).
```

Comment:
The support is not "integrated"; this implies that you get it when you install, which is not true. I would rephrase this to say that it is now possible to do development in all of these editors using the O# plugin.